### PR TITLE
Fix #3237 Add prep ID to artifacts list included in an analysis

### LINKED ISSUE #3237: Add prep ID to list of artifacts included in an analysis
Hello,

I think it would be useful to include the prep ID for the artifacts listed as included in an analysis. This will make it easier to track back to individual artifacts.

Thank you!

### PR DESCRIPTION
**Pull request by charles-cowart**
Updated UI and API call to return a list of artifact.prep_template ids included in each analysis listed.

### CODE REVIEW

File: qiita_pet/handlers/analysis_handlers/base_handlers.py [hunk 1]
```diff
@@ -74,9 +74,10 @@ def analysis_description_handler_get_request(analysis_id, user):
     artifacts = {}
     for aid, samples in analysis.samples.items():
         artifact = Artifact(aid)
+        foo = list(set([str(x.id) for x in artifact.prep_templates]))
```
Comment (antgonza) [inline qiita_pet/handlers/analysis_handlers/base_handlers.py:77]: Can you use a better name for the "foo" variable? Maybe prep_ids? Also, not sure why we need to convert to list, a set should be fine, no?

**Review by antgonza — CHANGES_REQUESTED** [review]
One comment. Also, could you copy/paste a screenshot of how this looks (the new analysis list) in your local Qiita install? Thank you.

File: qiita_pet/handlers/analysis_handlers/base_handlers.py (hunk 1 above)
Comment (charles-cowart) [reply qiita_pet/handlers/analysis_handlers/base_handlers.py]: doh! Thanks for the catch. :)

**Comment by charles-cowart** [comment]
prep_ids

**Comment by antgonza** [comment]
@charles-cowart, thank you. Waiting for CI to finish.

**Review by antgonza — APPROVED** [review]

**Pull request merged**